docs/89-NEWS.md

```diff
@@ -1,10 +1,186 @@
+existing user configurations.
+
+## New features
+
+ * `gears.string` now has a `endswith` and `startswith` functions
+ * `luarocks` modules are now automatically available in Awesome
+ * A generic way to create or use widgets have been added
+   (`wibox.widget.base.make_widget_from_value`)
+ * It is now possible to connect to signals from all instances of a widget at once
+ * The calendar widget now supports margins
+ * The documentation has a new theme
+ * Wiboxes now have a `to_widget()` and `save_to_svg()` methods.
+ * The client objects now have a `immobilized_horizontal` and
+   `immobilized_vertical` property to know if they can currently be moved or
+   resized (for example, it is set to false when maximized)
+ * `gears.timer` objects now have a `call_now` method.
+ * The hoykey popup now supports `termite` keybindings
+ * The menubar now loads faster
+ * The wiboxes now have an `input_passthrough` property to send mouse clicks
+   to the object below.
+ * The `taglist` and `tasklist` now support the declarative constructor syntax
+ * There is now an `awesome.pixbuf_to_surface` to bypass some LGI limitations
+ * The notifications icon can now be resized and limited with
+   `notification\_icon_\size`
+ * A `gears.sort` module has been added with graph resolution
+ * `awesome-client` now runs code in a protected context
+ * The <a href="./03-declarative-layout.md.html">widget documentation</a> has
+   been extended to be more friendly to new users.
+ * There is a new `beautiful.maximized_hide_border` theme option to hide the
+   border for maximized clients.
+ * The `client` `startup_id` field is now writable. This is useful when the
+   client native implementation is not present or too buggy to be used.
+ * The `awful.widget.prompt` now has a `with_shell` option to allow Bash/ZSH
+   aliases, function and environment variables to be used in commands.
+ * The `awful.titlebar`s now have a `fallback_name` when a client has no `name`
+   property.
+ * Clients now have a `motif_wm_hints` property to reflect some hints using the
+   Motif X11 property extension. This is used by some modern toolkits including
+   GTK.
+ * Clients now have a `requests_no_titlebar` property to expose when a client
+   has client side titlebars (also known as decorations and CSD)
+ * The hotkey popup now has a `show_awesome_keys` option.
+ * The `awful.widget.prompt` now has more of the `awful.prompt` constructor
+   arguments.
+ * It is now possible to set a list of layouts per tag instead of a single
+   global one.
+ * There is now a `awful.layout.get_tag_layout_index()` function to get the
+   index of the current layout in the global layout list
+   (`awful.layout..layouts`)
+
+### Better DPI handling
+
+The screen now has a read/write `dpi` property and `awful.screen.set_auto_dpi_enabled(true)`
+can be used to automatically set the DPI for many Awesome elements. Please
+note that it is not backward compatible and breaks many widget. As AwesomeWM
```

Comment:
```suggestion
note that it is not backward compatible and breaks many widgets. Since AwesomeWM
```

Reply:
`not backward compatible and will break many widgets.` ?